backend/src/main/java/org/example/post/domain/entity/PostEntity.java

```diff
@@ -28,6 +33,8 @@
 
 @Entity
 @Table(name = "post")
+@SQLDelete(sql = "UPDATE post SET removed_at = CURRENT_TIMESTAMP WHERE post_id=?")
+@Where(clause = "removed_at IS NULL")
```

Comment:
저희가 hinerate-core-6.5.2 버전 사용중이며, 6.3.0 이후부터 `@Where`가 Deprecated 되었다고 합니다.
`@SQLRestriction`으로 변경해야 합니다.
- https://docs.jboss.org/hibernate/orm/6.5/javadocs/org/hibernate/annotations/SQLRestriction.html
- https://devoong2.tistory.com/entry/JPA-Where-%EC%96%B4%EB%85%B8%ED%85%8C%EC%9D%B4%EC%85%98-%EC%82%AC%EC%9A%A9%EB%B2%95

```suggestion
@SQLRestriction("removed_at IS NULL")
```

Reply:
테스트 해본 후 잘 되면 다시 push 하겠습니다.